src/storyMap/components/StoryMapNew.js

```diff
@@ -0,0 +1,150 @@
+import React, { useCallback, useMemo } from 'react';
+
+import _ from 'lodash/fp';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+
+import { Paper, useMediaQuery } from '@mui/material';
+
+import PageContainer from 'layout/PageContainer';
+import PageHeader from 'layout/PageHeader';
+import { useAnalytics } from 'monitoring/analytics';
+import { ILM_OUTPUT_PROP, LANDSCAPE_NARRATIVES } from 'monitoring/ilm';
+import { useBreadcrumbsParams } from 'navigation/breadcrumbsContext';
+
+import { addStoryMap } from 'storyMap/storyMapSlice';
+
+import { MAPBOX_STYLE_DEFAULT } from 'config';
+
+import StoryMapForm from './StoryMapForm';
+import {
+  ConfigContextProvider,
+  useConfigContext,
+} from './StoryMapForm/configContext';
+
+import theme from 'theme';
+
+const BASE_CONFIG = {
+  style: MAPBOX_STYLE_DEFAULT,
+  theme: 'dark',
+  showMarkers: false,
+  use3dTerrain: true,
+  title: '',
+  subtitle: '',
+  byline: '',
+  chapters: [
+    // {
+    //   id: 'third-identifier',
+    //   alignment: 'left',
+    //   title: 'Chapter 1',
+    //   description: 'Copy these sections to add to your story.',
+    //   location: {
+    //     center: {lng: -78.55645099999998, lat: -0.16672202757398225},
+    //     zoom: 12.52,
+    //     pitch: 8.01,
+    //     bearing: 0.0,
+    //   },
+    //   mapAnimation: 'flyTo',
+    //   rotateAnimation: false,
+    //   callback: '',
+    //   onChapterEnter: [],
+    //   onChapterExit: [],
+    // },
+    // {
+    //   id: 'fourth-chapter',
+    //   alignment: 'right',
+    //   title: 'Chapter 2',
+    //   description: 'Copy these sections to add to your story.',
+    //   mapAnimation: 'flyTo',
+    //   location: {
+    //     center: {lng: -78.55645099999998, lat: -0.16672202757398225},
+    //     zoom: 4,
+    //     pitch: 0,
+    //     bearing: 0,
+    //   },
+    //   rotateAnimation: false,
+    //   callback: '',
+    //   onChapterEnter: [],
+    //   onChapterExit: [],
+    // },
```

Comment:
@paulschreiber I am not sure, since this is javascript there is no definition of what this structure is, so having it somewhere helps to understand how this looks, what do you suggest for this?

Reply:
suggestion: something along the lines of the following maybe so it's clear the purpose is to provide a scheme and the values don't matter?
```suggestion
    // chapter schema:
    // {
    //   id: string,
    //   alignment: 'left' | 'center' | 'right',
    //   title: string,
    //   description: string,
    //   location: {
    //     center: {lng: number, lat: number},
    //     zoom: number,
    //     pitch: number,
    //     bearing: number,
    //   },
    //   mapAnimation: ???,
    //   rotateAnimation: boolean,
    //   callback: ???,
    //   onChapterEnter: [???],
    //   onChapterExit: [???],
    // }
```

Reply:
Can we put this in README-type file, i.e. STORYMAP.md?